routes/authenticationRoute.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+// const { authenticateToken } = require("../middleware/authenticate");
+const authFunctions = require("../controller/authenticationController");
+const { authenticateToken } = require("../middleware/authenticate");
+
+const authenticateRouter = express.Router();
+authenticateRouter.post("/login", authFunctions.login);
+authenticateRouter.get("/refreshToken",authenticateToken,authFunctions.refreshToken);
```

Comment:
if authenticateToken is used here, for expired token it will not work the expected way
please remove authenticateToken here